src/crypto/convert.js

```diff
@@ -17,10 +17,35 @@ import ed2curve from 'ed2curve';
 import hash from './hash';
 
 /**
+ * @method bigNumberToBuffer
+ * @param {Number} bignumber
+ * @param {Number} size
+ *
+ * @return {Buffer}
+ */
+
+export function bigNumberToBuffer(bignumber, size) {
```

Comment:
Maybe beyond the scope of this PR, but a lot of these hyper-atomic functions would be way prettier as one-line arrow functions.